docs/prerequisites.md

```diff
@@ -15,6 +15,7 @@ The following prerequisites are required for installing OpenEBS.
    **Note:** Do not install open-iscsi / iscsi-initiator-utils on host nodes if a kubelet container, for example, Rancher Container Engine (RKE) already has the package installed.
 
 3. kubectl or helm installed and ready to use. Make sure you run the `kubectl` commands with cluster admin context. The installation will involve create a new Service Account and assigned to OpenEBS components.
```

Comment:
Make sure = Ensure that you run
....will involve creating a new Service Account that is assigned to OpenEBS components.